api/src/main/java/org/accula/api/detector/parser/Token.java

```diff
@@ -0,0 +1,28 @@
+package org.accula.api.detector.parser;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.jetbrains.annotations.NotNull;
+
+//Temporally unused class, will be used when generification of input data in CloneIndexer is being introduced
```

Comment:
If so, let's remove it. All we know that it's a bad practice to leave unused files